fhirzeug/generators/python_pydantic/templates/resource_header.py

```diff
@@ -123,6 +123,22 @@ def strip_empty_items(cls, valuse):
         """This strips all empty elements according to the fhir spec."""
         return _without_empty_items(valuse) or {}
 
+    @pydantic.root_validator(pre=True)
+    def validate_list_not_allowed_for_singleton_fields(cls, values):
```

Comment:
I'd add a TODO to remove it when Pydantic V2 is available and https://github.com/samuelcolvin/pydantic/issues/1268 is fixed.